### PR TITLE
SILA-2734: Allow empty response to not crash signer server

### PR DESCRIPTION
**Pull request by milind0samarth**

### CODE REVIEW

**Review by BenBlueAlvi — COMMENTED** [review]
The extra space on line 183 is unnecessary, but other than that looks good!

File: signerserver/application.py [hunk 1]
```diff
@@ -180,10 +180,12 @@ def forward():
             "headers": dict(response.headers),
         }
         try:
+
             proxy_response_dict["response"]["content"] = response.content.decode('utf-8')
-            proxy_response_dict["response"]['json_content'] = response.json()
+            if proxy_response_dict["response"]["content"] != "":
```
Comment (pyscape) [inline signerserver/application.py:185]: Why not:
```
if not proxy_response_dict["response"]["content"]:
```
It seems like you're covering more exceptions with the above.

File: signerserver/application.py [hunk 2]
```diff
@@ -180,10 +180,12 @@ def forward():
...
+                proxy_response_dict["response"]['json_content'] = response.json()
```
Comment (pyscape) [inline signerserver/application.py:186]: Should None always be returned as JSON? What if there is an actual text/html response that is empty?

Comment (milind0samarth) [reply signerserver/application.py:186]: This condition prevents exception from being sent on Empty response.

File: signerserver/application.py (hunk 1 above)
Comment (milind0samarth) [reply signerserver/application.py:185]: Exception is thrown if the content is "". We are preventing the exception being thrown

Comment (pyscape) [reply signerserver/application.py:185]: What about when proxy_response_dict["response"]["content"] is None?

Comment (pyscape) [reply signerserver/application.py:185]: proxy_response_dict["response"]["content"] is never None. Got it.

**Review by pyscape — APPROVED** [review]
Looks GTG

**Pull request merged**